pkg/asset/installconfig/aws/availabilityzones.go

```diff
@@ -6,11 +6,12 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
+	awstypes "github.com/openshift/installer/pkg/types/aws"
```

Comment:
i think the linter is going to want the installer package in its own grouping

Reply:
Thanks. yeah, I realized many other lint and fmt issues after rebase. I am addressing this today.